doc/man3/OSSL_CMP_SRV_CTX_new.pod

```diff
@@ -91,6 +92,8 @@ OSSL_CMP_SRV_CTX_set_grant_implicit_confirm
  int OSSL_CMP_SRV_CTX_set_accept_raverified(OSSL_CMP_SRV_CTX *srv_ctx, int val);
  int OSSL_CMP_SRV_CTX_set_grant_implicit_confirm(OSSL_CMP_SRV_CTX *srv_ctx,
                                                  int val);
+ int OSSL_CMP_SRV_CTX_centralKeygen_req(const OSSL_CRMF_MSG *crm,
+                                        const X509_REQ *p10cr);
```

Comment:
Actually this function should be named `OSSL_CRMF_MSG_centralKeygen_requested`.
So please rename it and move its doc to, e.g., `OSSL_CRMF_MSG_get0_tmpl.pod`.
And its header should consequently move from `cmp.h.in` to `crmf.h.in`.